### ISSUE TITLE
Add an option to preserve documentation comments

### ISSUE THREAD

**Issue by dalexand** [brigand/babel-plugin-flow-react-proptypes#137]
Please add an option to preserve documentation comments in output proptypes.

An input example:
 
```
export type ComponentProps = { 
    /**
     * Filter Metadata
     */
    filter: object;
};
```

An output:
```
ComponentProps.propTypes = {
    /**
     * Filter Metadata
     */
    filter: require('prop-types').object
}
```

**Comment by dalexand** [comment]
There is no way to write comments with babel-type. Required api will be added in babel 7. 
See [https://github.com/babel/babel/issues/5897](url)

**Comment by brigand** [comment]
Good idea, I think it's the best default behavior. Published in 7.0.0.

It only works for comments directly before object type properties, so e.g. `filter: /* x */ object` would lose the comment, but your examples would preserve it.
